ztp/siteconfig-generator/siteConfig/clusterCRsV1.go

```diff
@@ -179,4 +179,15 @@ spec:
     enabled: true
   searchCollector:
     enabled: false
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
```

Comment:
Why is it necessary to create a NetworkPolicy for the cluster namespace on the hub cluster?